### ISSUE TITLE
WBP-DPO logical alignment review

### ISSUE THREAD

**Issue by matentzn** [obophenotype/c-elegans-phenotype-ontology#141]
@Clare72 (FlyBase) has kindly started a review of alignments between DPO and WBP. Here are her notes:

DPO | WBPHENOTYPE | Label DPO | Label WBP | Alignment | Note
-- | -- | -- | -- | -- | --
FBcv_0000408 | WBPhenotype_0000067 | stress response defective | organism stress response variant | logical | does not need to be organism-wide
FBcv_0000409 | WBPhenotype_0000876 | osmotic stress response defective | organism osmotic stress response variant | logical | does not need to be organism-wide
FBcv_0000410 | WBPhenotype_0001273 | heat stress response defective | organism heat response variant | logical | does not need to be organism-wide
FBcv_0000683 | WBPhenotype_0000146 | temperature response defective | organism temperature response variant | logical | does not need to be organism-wide
FBcv_0000725 | WBPhenotype_0001620 | oxidative stress response defective | organism oxidative stress response variant | logical | does not need to be organism-wide

@chris-grove What do you think? Thanks @Clare72!!

**Comment by chris-grove** [comment]
@matentzn @Clare72
Thanks!
I'll need to look this over in more detail at some point when I get the time. One question is: what does it mean for the stress response to be "defective"? Would increased stress resistance also be considered "defective stress response"? The WBP "stress response variant" terms include (are parents or ancestors of) stress resistance increased terms. The "organism" environment response terms above for WBP are not, according to the logical definition or textual definition, necessarily organism-wide. We do have some cell stress response terms which are specific to a cell's response to stress (more specifically than just the whole organism).

**Comment by matentzn** [comment]
It depends on how you understand the word defective. I always understood it as a synonym to "abnormal" - so yes, abnormally increased stress response is a defective stress response..

**Comment by Clare72** [comment]
Yes, in dpo 'defective' is synonymous with 'abnormal' - any deviation from control/wild-type. I am considering changing our term labels at some point if all other phenotype ontologies prefer 'abnormal' and if 'defective' is causing confusion.

Ok, if "organism" in the labels does not mean "organism-wide" then I think these phenotypes are correctly matched, just wanted to flag it to check.

**Comment by chris-grove** [comment]
In WBP we have terms for "variant/abnormal" and separately subclass terms that have "defective". I think to worm biologists, "defective" means unable to function or complete the process or do so as effectively as wild type/control/normal. So, we wouldn't consider increased efficacy as "defective". This comes back to the whole "defective" pattern I was trying to create before and I still need to address our "defective" terms and finding appropriate patterns for them.
